meson_options.txt

```diff
@@ -0,0 +1,219 @@
+    description : 'link args for libbz2')
+
+option(
+    'boost_link_args', 
+    type : 'array', 
+    value : [
+        '-L/usr/lib64',
+        '-lboost_system',
+        '-lboost_context',
+        '-lboost_thread'], 
+    description : 'link args for boost')
+
+option(
+    'brotli_link_args',
+    type : 'array',
+    value : [
+        '-L/usr/local/lib',
+        '-lbrotlienc',
+        '-lbrotlidec'],
+    description : 'link args for libbrotli')
+
+option(
+    'bz2_link_args',
+    type : 'array',
+    value : [
+        '-L/usr/local/lib',
+        '-lbz2'],
+    description : 'link args for libbz2')
+
+option(
+    'curl_link_args',
+    type : 'array',
+    value : [
+        '-L/usr/local/lib',
+        '-lcurl'],
+    description : 'link args for libcurl')
+
+option(
+    'dl_link_args',
+    type : 'array',
+    value : [
+        '-L/usr/local/lib',
+        '-ldl'],
+    description : 'link args for libdl')
+
+option(
+    'editline_link_args',
+    type : 'array',
+    value : [
+        '-L/usr/local/lib',
+        '-leditline'],
+    description : 'link args for editline_link_args')
+
+option(
+    'lowdown_link_args', 
+    type : 'array', 
+    value : [
+        '-llowdown'], 
+    description : 'link args for lowdown')
+
+option(
+    'lzma_link_args',
+    type : 'array',
+    value : [
+        '-L/usr/local/lib',
+        '-llzma'],
+    description : 'link args for liblzma')
+
+option(
+    'openssl_link_args',
+    type : 'array',
+    value : [
+        '-L/usr/local/lib',
+        '-lcrypto'],
+    description : 'link args for openssl')
+
+option(
+    'pthread_link_args',
+    type : 'array',
+    value : [
+        '-L/usr/local/lib',
+        '-pthread'],
+    description : 'link args for pthread')
+
+option(
+    'sodium_link_args',
+    type : 'array',
+    value : [
+        '-L/usr/local/lib',
+        '-lsodium'],
+    description : 'link args for libsodium')
+
+option(
+    'sqlite3_link_args',
+    type : 'array',
+    value : [
+        '-L/usr/local/lib',
+        '-lsqlite3'],
+    description : 'link args for sqlite3')
+
+option(
+    'gc_link_args',
+    type : 'array',
+    value : [
+        '-L/usr/local/lib',
+        '-lgc'],
+    description : 'link args for boehm garbage collector')
+
+option(
+    'zlib_link_args', 
+    type : 'array', 
+    value : [
+        '-lz'], 
+    description : 'link args for zlib')
+
+
+# optional dependencies
+#============================================================================
+
+option(
+    'with_gc',
+    type : 'boolean',
+    value : 'true',
+    description : 'Build nix with Boehm garbage collector')
+
+option(
+    'with_libsodium',
+    type : 'feature',
+    value : 'auto',
+    description : 'Build nix with libsodium')
+
+option(
+    'with_s3',
+    type : 'boolean',
+    value : 'false',
+    description : 'Build nix with s3')
+
+option(
+    'with_coreutils_bin',
+    type : 'string',
+    description : 'path of cat, mkdir, etc.')
```

Comment:
generally if you want to override this, you would use a machine file (cross or native file) with the binaries section:
```ini
[constants]
binutils_root = '/path/to/root/'

[binaries]
cat = binutils_root + 'cat'
mkdir = binutils_root = 'mkdir'
...
```

Reply:
true! This was only added to mirror the existing buildsystem